src/css/layout/inner.scss

```
@@ -1,2 +1,619 @@
 @use '../object/utility/utility.scss';
-// innerのスタイルを記述するファイル
\ No newline at end of file
+@use '../object/component/button.scss';
+// innerのスタイルを記述するファイル
+.l {
+  &-inner {
```

[nits]
`l`と`-inner`は&で区切らなくて大丈夫です！

```suggestion
  .l-inner {
```

---

修正いたしました。
3affbf9d2e9ddff9530dc232f1126230ec68ff88